stream_alert_cli/terraform_generate.py

```diff
@@ -303,6 +316,8 @@ def generate_kinesis(cluster_name, cluster_dict, config):
         's3_logging_bucket': logging_bucket
     }
 
+    return True
```

Comment:
I'm doing it for consistency, this could change in the future as we add more checks for validity